go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/go-logr/logr v0.2.1-0.20200730175230-ee2de8da5be6
 	github.com/go-logr/zapr v0.2.0 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
We already have 2 logging packages. Do we need this new one?

Reply:
I think this one can be removed.

Reply:
zapr is indirect so glog would be the 2nd actually. I found logr to be tricky while using file level logging vs glog. So, just didn't spend enough time on it. Object level logging using logr in controller_provisioning.go was a bit simpler.

Reply:
Imported controller-runtime for its logger so we don't pull in glog as a new logging package.